tscreen_linux.go

```diff
@@ -76,12 +78,11 @@ func (t *tScreen) termioInit() error {
 	// TCSETS.  This can leave some output unflushed.
 	ioc = uintptr(syscall.TCSETS)
 	if _, _, e1 := syscall.Syscall6(syscall.SYS_IOCTL, fd, ioc, tios, 0, 0, 0); e1 != 0 {
+		log.Printf("HUH %v", e1)
 		e = e1
 		goto failed
 	}
 
-	signal.Notify(t.sigwinch, syscall.SIGWINCH)
```

Comment:
was this intentional? this line is still there for the other arches

Reply:
oh huh. this was DB code, i think it's probably accidental, will put back.